frontend-react/src/pages/admin/value-set-editor/ValueSetsDetail.tsx

```diff
@@ -105,9 +105,8 @@ const saveData = async (
         throw new Error("A null row was encountered in saveData");
     }
 
-    const endpointHeaderUpdate = lookupTableApi.saveTableData<ValueSetRow[]>(
-        LookupTables.VALUE_SET_ROW
-    );
+    const endpointHeaderUpdate =
```

Comment:
based on what I'm seeing I think we can clean up this file a bit more. 

Can we:

* remove the dependency on sending up the entire table on save? see https://github.com/CDCgov/prime-reportstream/blob/14ece8856c01d7e979bfa5f72c484e2aa21bb983/frontend-react/src/pages/admin/value-set-editor/ValueSetsDetail.tsx#L113
* remove the need to filter all data based on the name of the data set we want to display? see https://github.com/CDCgov/prime-reportstream/blob/14ece8856c01d7e979bfa5f72c484e2aa21bb983/frontend-react/src/pages/admin/value-set-editor/ValueSetsDetail.tsx#L158

Reply:
For the first comment: we will _still_ need to save/send up the entire table. Unfortunately, that is still a limitation of the lookuptable api. The nice thing about that now is that the tables we're saving are muuuuch smaller than sender_automation_table_row. Except that patient_preferred_langage one, it's still a beast, but whatcha gonna do.

For the second comment: absolutely! no need to filter anymore.

Reply:
@sethdarragile6 So the POST request to update table data will go to `{api_root}/{tableName}` ex. `http://localhost:3000/api/lookuptables/patient_preferred_language` instead of `{api_root}/sender_automation_value_set_row` but we will still send up ALL rows? That doesn't really make sense to me - not sure why the endpoint would change if the API is still expecting all rows

Reply:
@doug-s-nava maybe we should zoom about this, but yes- we are still sending up ALL rows in whatever particular table we're editing (`patient_preferred_language`, in this case). The only real change in the POST endpoint request should be swapping out the tablename `sender_automation_value_set` for the tablename `patient_preferred_language` (or `gender` or `yesno` etc). 

As far as I know, the POST endpoint still expects that all rows for the specified table are being sent. If you were to only send the particular item you edited, for example, you would wind up updating the table so that it had ONLY that row in it. Hilarious! (not so much)

Reply:
ok, just wanted to make sure. I'm assuming that the back end is then doing some sort of filter based on the path segment (ex. `patient_preferred_language`)? Just trying to be clear on why that endpoint segment is changing if we still need to send up the entire table